pyro/infer/elbo.py

```diff
@@ -15,6 +23,11 @@ class ELBO(object):
         :func:`pyro.iarange` contexts. This is only required to enumerate over
         sample sites in parallel, e.g. if a site sets
         ``infer={"enumerate": "parallel"}``.
+    :param bool vectorize_particles: Vectorize ELBO computation over `num_particles`.
```

Comment:
consider adding more info:
"Whether to vectorize the ELBO computation over `num_particles`. Defaults to False. This requires static structure in model and guide."

Reply:
Thanks, will update.